### DISCUSSION TITLE
Hourly data for more than 7 days

### DISCUSSION THREAD

**Discussion by beow** [opennem/opennem#312] [General]
I have a simple balance model (in python/pandas) that can simulate different power system scenarios. Input for the model is hourly wind/solar power data for a year or longer. Wind/solar is treated, together with other power sources, as "must-run" sources that are balanced against consumption with flexibility sources (storage/hydro/import/export etc) with a given "merit-order". See for example https://adelsfors.se/2023/06/19/sweth-and-swenth-model-for-germany/ 

I would very much like to model the Western Australia (SWIS) power system with my model. It is interesting since it does not have the "import/export degree of freedom" and is hence simpler to model. But for this I need hourly wind/PV data as given by the export CSV button on https://opennem.org.au/energy/wem/?range=7d&interval=30m , but for a calendar year (or two). When using the web page for for example 1Y the export CSV resolution goes to weekly resolution, as in the graph. 

Would be very happy if the OpenNEM community could help out with providing this data somehow.

**Comment by aleith** [comment]
this has been on the FR list for many years :-) maybe it isn't planned but I was told a lot of backend and frontend work was required before this would be possible to add to the website. there's also the openNEM python tools which can fetch this data for you.
 

there's a new public portal that University of Victoria put online which can provide this resolution of data and provides many filtering options too, both for displaying the data and downloading it.

**Comment by aleith** [comment]
see [#22 ](https://github.com/opennem/opennem-fe/issues/22) Navigating dataset by time (with cursor, shortcuts or date modification)
and #316 FE energy charts to be able to display historical data down to 5 min resolution
